cinema/views.py

```diff
@@ -87,6 +111,8 @@
         )
     )
     serializer_class = MovieSessionSerializer
+    authentication_classes = (TokenAuthentication,)
```

Comment:
Consider adding `SessionAuthentication` along with `TokenAuthentication` for more flexibility in authentication methods, similar to the suggestion for other viewsets.